index.html

```diff
@@ -12,291 +12,250 @@
-                <div class="left">
-                    <div class="content">
-                        <div class="h4">Travel</div>
-                        <div class="p">The best travel agent in the world. Contact us immediatly now!</div>
-
-                        <div class="social-links">
-                            <a href="#">
-                                <img src="Images/IG.png" alt="">
-                            </a>
-                            <a href="#">
-                                <img src="Images/twitter.png" alt="">
-                            </a>
-                            <a href="#">
-                                <img src="Images/YT.png" alt="">
-                            </a>
-                        </div>
+            <img src="Images/footer-divider.png" alt="" class="divider">
+        </div>
+        <div class="travel-footer">
+            <div class="left">
+                <div class="content">
+                    <div class="h4">Travel</div>
+                    <div class="p">The best travel agent in the world. Contact us immediatly now!</div>
+                    <div class="social-links">
+                        <a href="#">
+                            <img src="Images/IG.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="Images/twitter.png" alt="">
+                        </a>
+                        <a href="#">
+                            <img src="Images/YT.png" alt="">
+                        </a>
                     </div>
                 </div>
-
-                <div class="right">
-
-                    <div class="menu">
-                        <ul>
-                            <li>
-                                <a href="#">Home</a>
-                            </li>
-
-                            <li>
-                                <a href="#">Destinations</a>
-                            </li>
-
-                            <li>
-                                <a href="#">Guide</a>
-                            </li>
-
-                            <li>
-                                <a href="#">Contact Us</a>
-                            </li>
-                        </ul>
-                    </div>
+            </div>
+            <div class="right">
+                <div class="menu">
+                    <ul>
+                        <li>
+                            <a href="#">Home</a>
+                        </li>
+                        <li>
+                            <a href="#">Destinations</a>
+                        </li>
+                        <li>
+                            <a href="#">Guide</a>
+                        </li>
+                        <li>
+                            <a href="#">Contact Us</a>
+                        </li>
+                    </ul>
                 </div>
             </div>
         </div>
     </footer>
 </body>
-</html>
```

Comment:
Hi @Xfarouq,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    <img src="https://camo.githubusercontent.com/7cf1ba8db9cd08f454a9651e48128c064a4440a59881658c8593a41a8de243f3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966" alt="Remote Collaboration" style="width:40%;">
</div>

###  Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Correctly Applied the use of Html Semantics** 👍
- **Good Commit Message**  👍
- **Descriptive Pull Request** 👍

### Required Changes ♻️
- Please improve your page UX to avoid overflow